lib/std/Thread.zig

```diff
@@ -76,6 +76,34 @@ pub fn spinLoopHint() void {
     }
 }
 
+pub fn setName(self: Thread, name: [:0]const u8) !void {
+    if (use_pthreads) {
```

Comment:
Early exit:
```zig
if (!use_pthreads)
    return;
```

And a pure Zig implementation for Linux can be implemented using musl's `pthread_setname_np` as guide.

Reply:
ok for the early exit. and yeah I planned to follow musl's implementation.